### PR TITLE
Adding fixture for GiftCardConfiguration

### LINKED ISSUE #81: Create fixture for gift card configuration
When this fixture is made, add it to the test application

### PR DESCRIPTION
**Pull request by Roshyo**
Fixes #81

### CODE REVIEW

File: src/Fixture/Factory/GiftCardConfigurationExampleFactory.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            $giftCardConfiguration = $this->giftCardConfigurationFactory->createNew();
+        }
+
+        $giftCardConfiguration->setCode($options['code']);
+        $giftCardConfiguration->setEnabled($options['enabled']);
+        $giftCardConfiguration->setDefault($options['default_configuration']);
+
+        $imagePath = $options['background_image'];
+
+        $imagePath = $this->fileLocator === null ? $imagePath : $this->fileLocator->locate($imagePath);
```
Comment (loevgaard) [inline src/Fixture/Factory/GiftCardConfigurationExampleFactory.php:78]: `$this->fileLocator` is never null?

File: src/Fixture/GiftCardConfigurationFixture.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    protected function configureResourceNode(ArrayNodeDefinition $resourceNode): void
+    {
+        $resourceNode
+            ->children()
+                ->scalarNode('code')->cannotBeEmpty()->end()
+                ->scalarNode('background_image')->cannotBeEmpty()->end()
+                ->booleanNode('enabled')->end()
+                ->booleanNode('default_configuration')->end()
```
Comment (loevgaard) [inline src/Fixture/GiftCardConfigurationFixture.php:24]:
```suggestion
                ->booleanNode('default')->end()
```

Probably simpler?

File: src/Fixture/Factory/GiftCardConfigurationExampleFactory.php (hunk 1 above)
Comment (Roshyo) [reply src/Fixture/Factory/GiftCardConfigurationExampleFactory.php]: C/P gone wrong :D

File: src/Fixture/GiftCardConfigurationFixture.php (hunk 2 above)
Comment (Roshyo) [reply src/Fixture/GiftCardConfigurationFixture.php]: Definitely

**Pull request merged**